tests/unit/views/CartView.spec.ts

```diff
@@ -1,6 +1,9 @@
-import { mount, createLocalVue, RouterLinkStub } from '@vue/test-utils'
+import { mount, createLocalVue, RouterLinkStub, shallowMount } from '@vue/test-utils'
+import mutations from '@/store/mutations'
+// @ts-ignore
+import { get } from '@molgenis/molgenis-api-client'
```

Comment:
if you move this type of code to the actions you do not need to duplicate the test setup

Reply:
CartView.spec.ts went from 218 lines to 158. Where is the code duplication?